CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing
+
+Thank you for contributing to OpenShift Ansible. This document explains how the
+repository is organized, and how to submit contributions.
+
+## Introduction
+
+Before submitting code changes, get familiarized with these documents:
+
+- [Core Concepts](https://github.com/openshift/openshift-ansible/blob/master/docs/core_concepts_guide.adoc)
+- [Best Practices Guide](https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc)
+- [Style Guide](https://github.com/openshift/openshift-ansible/blob/master/docs/style_guide.adoc)
+
+## Repository structure
+
+### Scripts
+
+```
+.
+└── bin                 Contains the `bin/cluster` script, a wrapper around the
+                        Ansible playbooks that ensures proper configuration, and
+                        facilitates installing, updating, destroying and
+                        configuring OpenShift clusters.
```

Comment:
Hmm, alright, I see `bin/` hasn't got any code changes since Sep 29th (apart from typo fixes I made recently).

I don't really know what's the status of that script, if deprecated/unmaintained, we/I can remove it or any proper notes.